tests/test_builders/test_build_epub.py

```diff
@@ -34,35 +41,41 @@ class EPUBElementTree:
         'epub': 'http://www.idpf.org/2007/ops',
     }
 
-    def __init__(self, tree):
+    def __init__(self, tree: ET.Element) -> None:
         self.tree = tree
 
     @classmethod
-    def fromstring(cls, string):
+    def fromstring(cls, string: str | bytes) -> Self:
         tree = ET.fromstring(string)  # NoQA: S314  # using known data in tests
         return cls(tree)
 
-    def find(self, match):
+    def find(self, match: str) -> Self:
         ret = self.tree.find(match, namespaces=self.namespaces)
-        if ret is not None:
-            return self.__class__(ret)
-        else:
-            return ret
```

Comment:
We never hit this branch, and keeping it meant that every use of `find` we might have to `assert ... is not None` to use the result as a `EPUBElementTree`.

Reply:
`assert` makes more sense than `cast` in a test, so i think this approach is good